src/components/organisms/loginForm/loginForm.tsx

```diff
@@ -1 +1,77 @@
-import React from 'react';
\ No newline at end of file
+import React from 'react';
+import InputText from '../../atoms/inputs'
+import ButtonComponent from '../../atoms/button'
+import {Form} from 'semantic-ui-react'
+import { action } from '@storybook/addon-actions';
+import { StyledFormField } from './style/loginForm.styled.component';
+
+
+export type loginFormInterface = {
+    isDisabled:boolean,
+    isPrimary:boolean,
+    isLoading:boolean,
+    isError?:boolean,
+}
+
+const onHandleSubmit = (event:any) => {
+    console.log(event,event.target.elements.email.value);
+    action('onSubmit')
+}
```

Comment:
I think this function is better made as a prop, especially because this is only an organism component. Would you agree?

Reply:
> I think this function is better made as a prop, especially because this is only an organism component. Would you agree?

Okay, I agree also in this.